arlo-client/src/components/AuditFlow/index.tsx

```diff
@@ -75,18 +60,19 @@ const AuditFlow: React.FC<IProps> = ({
     updateAudit()
   }, [updateAudit])
 
-  const board:
-    | IAuditBoard
-    | undefined = audit.jurisdictions[0].auditBoards.find(
-    (v: IAuditBoard) => v.id === token
-  )
+  const board: IAuditBoard | undefined = audit
```

Comment:
The type annotations in this statement are not required. TypeScript can figure them out. But leave them in if you like!